src/client/doc.h

```diff
@@ -258,4 +258,23 @@ constexpr const char* GmGetR = R"mydelimiter(
             dict
         )mydelimiter";
 
+constexpr const char* GmGetP = R"mydelimiter(
+        get_cga_prints() -> str
+
+        Returns the CGA print operations called during the generation of the model.
```

Comment:
"Returns a string with all CGA print output of this model separated by new line."